src/mpi/comm/commutil.c

```diff
@@ -310,10 +310,6 @@ static int MPIR_Comm_commit_internal(MPIR_Comm * comm)
     mpi_errno = MPID_Comm_create_hook(comm);
     MPIR_ERR_CHECK(mpi_errno);
 
-    /* Create collectives-specific infrastructure */
```

Comment:
@yfguo can we not delete this comment? When moving this call to the new location? I think it is still helpful.

Reply:
This comment is now out of place. Please move it to the new location of MPIR_Coll_comm_init